message_app/model.py

```diff
@@ -61,3 +69,107 @@ def get_last_user_id(cls) -> int:
         # Get last user ID
         id = User.query.order_by(User.id.desc()).first().id
         return id
+
+# SQLAlchemy model for conversation_request table
+class ConversationRequest(db.Model):
+    __tablename__ = 'conversation_request'
+    id = db.Column(db.Integer, primary_key=True)
+    initiator_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+    receiver_id = db.Column(db.Integer, db.ForeignKey("users.id"), nullable=False)
+    accepted = db.Column(db.Integer, nullable=False, default=0)
+    request_time = db.Column(db.Integer, nullable=False)
```

Comment:
The default value can be [time.time().](https://www.tutorialspoint.com/python/time_time.htm)

Reply:
not really, as a request might timeout and in that case, we should not even create the record at all.